fakenet/listeners/ProxyListener.py

```diff
@@ -0,0 +1,397 @@
+import socket
+import SocketServer
+import threading
+import sys
+import glob
+import time
+import importlib
+import Queue
+import select
+import logging
+import ssl
+from OpenSSL import SSL
+from ssl_utils import ssl_detector 
+import hexdump
+import listeners
```

Comment:
Getting a module path error when installed using setup.py

  File "c:\program files\python27\lib\site-packages\fakenet_ng-1.2-py2.7.egg\fak
enet\listeners\ProxyListener.py", line 15, in <module>
    import listeners
ImportError: No module named listeners